backend/dev-process.md

```diff
@@ -9,28 +9,59 @@ parent: Backend
 
 ## Checklist for a new PR
 
-- [ ] If the change is linked to an issue present in the [Planning](https://github.com/orgs/France-ioi/projects/2),
-set its status to `in progress`
-- [ ] If it addresses an issue, make sure the issue is linked in the `Development` tab on the right.
-This will allow the issue to be closed automatically when the commit is merge,
-and update its status if it is present in the planning.
-- [ ] Add a description of what the PR is about
-- [ ] If choices were made, describe the reason you made them and the alternatives you considered with a trade-off analysis.
+### If the change is linked to an issue present in the [Planning](https://github.com/orgs/France-ioi/projects/2):
 
+- [ ] Set the issue's status to "in progress" in the planning.
 
-- **If a change is related to a service:**
-  * [ ] Make sure the swagger documentation is properly updated
-  * [ ] the description renders correctly
-  * [ ] all the field and their annotations are correct for the request and response
+![Change the status to "in progress"]({{ site.url }}{{ site.baseurl }}/assets/dev-process-status-update.png)
 
-Launch local:
-`swagger generate spec --scan-models -o ./swagger.yaml && swagger validate ./swagger.yaml && swagger serve ./swagger.yaml`.
+### If the PR fully resolves the issue:
 
-**Small tip:** You need to have two empty lines between paragraphs, otherwise the generated documentation will have everything on the same line.
+- [ ] Make sure the issue is linked in the "Development" tab on the right page of the PR. This will allow the issue to be closed automatically when the commit is merge, and update its status if it is present in the planning. You can also start the description of the PR with "fixes #issue_number" to link it automatically, but you should check that it worked.
```

Comment:
Not necessarily "start" the description with it, it can be anywhere.
"you should check that it worked." -> it always works with this technique

Reply:
fixed.